src/_pytest/runner.py

```diff
@@ -190,10 +191,11 @@ def check_interactive_exception(call, report):
 def call_runtest_hook(item, when, **kwds):
     hookname = "pytest_runtest_" + when
     ihook = getattr(item.ihook, hookname)
+    reraise = (Exit,)
+    if not item.config.getvalue("usepdb"):
+        reraise += (KeyboardInterrupt,)
     return CallInfo.from_call(
-        lambda: ihook(item=item, **kwds),
-        when=when,
-        reraise=KeyboardInterrupt if not item.config.getvalue("usepdb") else (),
+        lambda: ihook(item=item, **kwds), when=when, reraise=reraise
```

Comment:
Worth a `# fmt: off` / `# fmt: on` block maybe to silence black here?  (if this code stays like this)

Reply:
IMHO that's fine 😁